index.js

```diff
@@ -58,8 +65,15 @@
         }
       }(this.getDOMNode(), this.handleClickOutside));
 
-      document.addEventListener("mousedown", fn);
-      document.addEventListener("touchstart", fn);
+      if (document.addEventListener) {
+        // W3C events
+        document.addEventListener("mousedown", fn);
+        document.addEventListener("touchstart", fn);
+      } else if (document.attachEvent) {
+        // Old IE
+        document.attachEvent("onmousedown", fn);
```

Comment:
I'm not particularly interested in supporting IE8 or below, which are the only versions of IE where attachEvent is necessary. If you are on one of these versions of IE, you are also on an operating system that isn't even supported by Microsoft anymore, so in this case I will not accommodate those setups. They'll just have to get an OS that's less than 15 years old and abandoned by the company that used to support it.